SofascoreAcademy/SofascoreAcademy/Extensions/String+extension.swift

```diff
@@ -10,8 +10,19 @@ import Foundation
 extension String {
 
     static let emptyString: String = ""
+    
+    static let settings: String = "Settings"
+    static let eventDetails: String = "Event Details"
+    
+    static let football: String = "Football"
+    static let basketball: String = "Basketball"
+    static let americanFootball: String = "Am. Football"
+    
     static let finishedEvent: String = "FT"
     static let notStartedEvent: String = "-"
```

Comment:
Stringove bi trebalo razvrstati. 
Imati posebno stringove koje prikazuješ kao text na UI-u. To je ovo gore. To ti može biti u String+Common